website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
+Recently Viewed Products
+========================
+
+Let the users keep track of the products they saw on the ecommerce.
+Uses the session id as a key to store the product history, so it works
+for both authenticated and anonymous users
+
+Usage
+=====
+
+Open some products pages on the e-commerce, then hover or click on
+'My Products' in the top menu. You should see a page or a popover (click
+vs hover) with your last product views.
+
+TODO / Roadmap
```

Comment:
Instead of "TODO / Roadmap", call it "Known issues / Roadmap", that it's the standard name